src/pages/evm/nfts/NftViewer.vue

```diff
@@ -113,19 +121,220 @@ function toggleVideoPlay(playOnly?: boolean) {
         videoElement.value.pause();
     }
 }
+
+function calculateHoverPreviewPosition() {
+    const root = rootElement.value;
+    const iconContainerElement = iconElement.value;
+    if (!root || !iconContainerElement) {
+        return;
+    }
+
+    // the height of the chevron on the hover element that points to the root element
+    const chevronHeight = 14;
+    // the minimum padding between the hover element and the border of the window / the edge of the left navbar
+    const padding = 16;
+    // the width of the left nav
+    const leftNavWidth = platformStore.__evm_nav_is_collapsed ? 0 : 300;
```

Comment:
Is it okay to directly access the variable this way or will we use a getter?

Reply:
yeah you can access state directly, thats fine. getters are just for more complex ways of accessing, like if we need a label or something